docs/upgrade/upgrade-instructions-for-read-model-for-product-lists.md

```diff
@@ -5,6 +5,10 @@ Besides better logical separation of the application, it is a first step towards
 The read model package is marked as *experimental* at the moment so there is a possibility we might introduce some BC breaking changes there.
 You do not need to perform the upgrade instantly, however, if you do so, you will be better prepared for the upcoming changes.
 
+Right now there are two implementations of read model.
```

Comment:
The paragraph doesn't sound good for me, I suggest to rephrase it:

There are two implementations of read model facades right now as we want to provide you possibilities that suits you the best. You can choose between SQL or Elasticsearch implementation, please learn more in [Front-end product filtering](/docs/model/front-end-product-filtering.md).